README.md

```diff
@@ -15,7 +15,7 @@ A GitHub action to turn a GitHub project into a self-hosted Helm chart repo, usi
 
 For more information on inputs, see the [API Documentation](https://developer.github.com/v3/repos/releases/#input)
 
-- `version`: The chart-releaser version to use (default: v1.1.1)
+- `version`: The chart-releaser version to use (default: v1.1.0)
```

Comment:
I don’t get why change this version, the default cart-releaser version used is the v1.1.1, is only the chart-releaser-action version used that was wrong because it wasn’t released already, or I’m missing something?